src/LogOutputManager.ts

```diff
@@ -39,7 +39,7 @@ export class LogOutputManager {
         this.includeRegex = null;
         this.logLevelRegex = null;
         this.excludeRegex = null;
-        this.pkgRegex = /(\w+:\/.*\.(?:brs|xml))[ \t]*(?:\((\d+)(?:\:(\d+))?\))?/;
+        this.pkgRegex = /(?:\s*at line (\d*) of file )*(?:(pkg:\/|file:\/\/)(.*\.(bs|brs|xml)))[ \t]*(?:(?:(?:\()(\d+)(?:\:(\d+))?\)?)|(?:\:(\d+)?))*/;
```

Comment:
Can you add a comment to this regex explaining the various formats that are supported? It's not clear why `at line (\d*) of file)` is included here. Perhaps a block comment with examples of what log formats will be captured by this regex.

Reply:
@georgejecook btw, the comments you added are perfect. Thanks!